### PR TITLE
intel-media-driver: update to 20.4.5.

### PR DESCRIPTION
**Pull request by travankor**

### CODE REVIEW

File: srcpkgs/libva/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libva'
 pkgname=libva
-version=2.9.1
-revision=2
+version=2.10.0
```
Comment (ericonr) [inline srcpkgs/libva/template:3]: Update libva-glx as well?

Comment (ericonr) [reply srcpkgs/libva/template:3]: And leave a comment here about keeping these two up to date with each other.

Comment (travankor) [reply srcpkgs/libva/template:4]: Thanks, I thought this was previously added.

File: srcpkgs/intel-media-driver/template [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'intel-media-driver'
 pkgname=intel-media-driver
-version=20.4.1
+version=20.4.5
```
Comment (ericonr) [inline srcpkgs/intel-media-driver/template:3]: There is a https://github.com/intel/media-driver/releases/tag/intel-media-21.1.0 release out, but I'm not sure it's stable, so going to skip it for now.

**Pull request merged**